Platform/TigerlakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -47,7 +47,7 @@ def gen_xml_file(stitch_dir, stitch_cfg_file, btg_profile, plt_params_list, plat
 
     tree = ET.parse(new_xml_file)
 
-    xml_change_list = stitch_cfg_file.get_xml_change_list (platform, plt_params_list)
+    xml_change_list = stitch_cfg_file.get_xml_change_list (platform, plt_params_list, res)
```

Comment:
Enable resiliency by default. Try build with -fd and see what happens.

Reply:
When in release mode - enable resiliency. When in debug mode - disable resiliency.